pages/ingredients.js

```diff
@@ -0,0 +1,37 @@
+export const defaultIngredients = [
```

Comment:
This file should **not** be inside of pages. 

You could create a folder called `src` or `data` and move this file inside this folder.